MEDIASOUP_PROTOCOL.md

```diff
@@ -86,7 +86,7 @@ Request:
   target: 'peer',
   id: 1111,
   direction: 'send', // 'send'/'recv'.
-  options: {},
+  options: {tcp: true}, // enable or disable ICE-TCP
```

Comment:
Please, add a ending dot into the comment line :)

Reply:
Also, there are other transport options. In fact, the `options` Object in the `createTransport` request matches the [TransportOptions](https://mediasoup.org/documentation/mediasoup-client/api/#Transport-TransportOptions) Object.